src/configuration/Configuration.cpp

```diff
@@ -19,18 +19,18 @@ nlohmann::json Configuration::json() const {
   config["debug"]                              = debug;
 
   if (method == Method::Heuristic) {
-    auto& heuristic                   = config["settings"];
-    heuristic["initial_layout"]       = ::toString(initialLayout);
-    heuristic["admissible_heuristic"] = admissibleHeuristic;
-    heuristic["consider_fidelity"]    = considerFidelity;
-    if (lookahead) {
-      auto& lookaheadSettings           = heuristic["lookahead"];
+    auto& heuristicJson             = config["settings"];
+    heuristicJson["heuristic"]      = ::toString(heuristic);
+    heuristicJson["initial_layout"] = ::toString(initialLayout);
+    if (lookaheadHeuristic != LookaheadHeuristic::None) {
+      auto& lookaheadSettings           = heuristicJson["lookahead"];
+      lookaheadSettings["heuristic"]    = ::toString(lookaheadHeuristic);
```

Comment:
This could be the perfect place to add some more metadata from the functions you marked as `maybe_unused`. It could be helpful to know whether the selected heuristic is admissible, tight, etc.

Reply:
Great idea, thank you!